README.md

```diff
@@ -35,11 +35,11 @@ A developer environment for pulp based off of the [Pulp OCI Images](https://gith
     Note, the `/src/` folder in the container is the parent folder containing the `oci_env` and all
```

Comment:
I think this path is changing right? Can you grep this file for any other updates? Apologies if I'm misunderstanding things here.

Reply:
The path in the container stays the same. The only thing that's changing is the path that gets mounted into the container. `/src/` in the container used to always map to `oci_env/../` on the host machine. Now it maps to `oci_env/../` by default, but you can also configure it by setting `SRC_PATH` in your .env configuration if you want to change it. This allows you to have one checkout for `oci_env` that lets you run multiple different `SRC_PATH`s. For example I can check out `oci_env` in `~/code/oci_env` and create another directory at `~/code/hub/` that has different versions of galaxy_ng checked out on different branches:

```
~/code/hub
├── env-insights
│   ├── ansible-hub-ui
│   └── galaxy_ng
├── env-standalone
│   ├── ansible-hub-ui
│   └── galaxy_ng
├── insights.compose.env
└── standalone.compose.env
```

In this example I have `SRC_DIR=~/code/hub/env-insights/` set in my `insights.compose.env` file and `SRC_DIR=~/code/hub/env-standalone/` in my `standalone.compose.env`. I was using this to test syncing between different versions of hub